client/landing/jetpack-cloud/sections/scan/main.tsx

```diff
@@ -97,7 +97,7 @@ class ScanPage extends Component< Props > {
 					} )
 				}
 			>
-				{ translate( 'Contact Support {{externalIcon/}}', {
+				{ translate( 'Contact support {{externalIcon/}}', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.